docs/resolve/relationships.rst

```diff
@@ -394,8 +406,6 @@ Now the function will work:
     var func = scope.Resolve<FactoryDelegate>();
     var obj = func(1, 2, "three");
 
-Another option you have is to use a :doc:`delegate factory, which you can read about in the advanced topics section <../advanced/delegate-factories>`.
-
 Should you decide to use the built-in auto-generated factory behavior (``Func<X, Y, B>``) and only resolve a factory with one of each type, it will work but you'll get the same input for all constructor parameters of the same type.
```

Comment:
This extra bit down here is now "under" the `RegisterGeneratedFactory` header but is about `Func<T,U,V>` relationship so it feels kinda weird, like it should pertain to `RegisterGeneratedFactory` but probably belongs up above the `Delegate Factories` header.